spec/features/admin/products_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe 'Admin - Products', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.